src/csharp/PesterTests/TestTests.cs

```diff
@@ -0,0 +1,24 @@
+namespace PesterTests
+{
+    using Pester;
+    using Xunit;
+
+    /// <summary>
+    /// Test the test class
+    /// </summary>
+    public class TestTests
+    {
+        public readonly Test test;
+        public TestTests() => test = new Test();
```

Comment:
Why not just initialize it in the field initializer? Or write it in the test?

Reply:
wrote it in the test.